tests/test_unit/test_cli/test_behavior.py

```diff
@@ -369,6 +368,113 @@ def test_dump_datamodel(
     assert len(child_loaded.properties) == 1
 
 
+def test_dump_datamodel_skip_global(
+    tmp_path: Path, toolkit_client_approval: ApprovalToolkitClient, env_vars_with_client: EnvironmentVariables
+) -> None:
+    output_dir = tmp_path / "tmp_dump"
+    default_space_args: dict[str, Any] = dict(is_global=False, last_updated_time=0, created_time=0)
+    default_view_args: dict[str, Any] = dict(
+        last_updated_time=1,
+        created_time=1,
+        name=None,
+        description=None,
+        implements=None,
+        writable=True,
+        used_for="node",
+        is_global=False,
+        filter=None,
+    )
+    default_prop_args: dict[str, Any] = dict(
```

Comment:
The use of `Any` is discurraged: https://cognitedata.slack.com/archives/CDJUS0FAB/p1742557756225569. Could we add some safer type annotations (also the case for other Any usages in this PR)?